retriever/lib/retrieval.js

```diff
@@ -18,27 +18,32 @@
 export async function retrieveFile(
   baseUrl,
   pieceCid,
+  request,
   cacheTtl = 86400,
   { signal } = {},
 ) {
   const url = getRetrievalUrl(baseUrl, pieceCid)
-  const response = await fetch(url, {
-    cf: {
-      cacheTtlByStatus: {
-        '200-299': cacheTtl,
-        404: 0,
-        '500-599': 0,
-      },
-      cacheEverything: true,
-    },
-    signal,
-  })
-  const cacheStatus = response.headers.get('CF-Cache-Status')
-  if (!cacheStatus) {
-    console.log(`CF-Cache-Status was not provided for ${url}`)
-  }
 
-  const cacheMiss = cacheStatus !== 'HIT'
+  const cacheKey = new Request(url, request)
```

Comment:
Since the `fetch` call is not using any additional information from the original `request`, I think we shouldn't be using the original request for constructing the cache key. In particular, we should ignore request headers provided by the FilBeam client.

Would the following work?

```js
const originReq = new Request(url)
const cacheLKey = originReq
// ...
const response = await fetch(originReq)
```